Vic2ToHoI4/Source/HOI4World/HOI4Ideology.cpp

```diff
@@ -173,7 +167,7 @@ void HoI4Ideology::outputModifiers(std::ofstream& file) const
 	file << "\t\tmodifiers = {\n";
 	for (auto modifier: modifiers)
 	{
-		file << "\t\t\t" << modifier.first << " = " << modifier.second << "\n";
+		file << "\t\t\t" << modifier.first << " " << modifier.second << "\n";
```

Comment:
mmhm most of these functions should go into separate file as before.

Reply:
Yep, that'll be part of a future fix-up in that file.